Week3/homework/2-step3.js

```diff
@@ -3,6 +3,27 @@
 function threeFive(startIndex, stopIndex, threeCallback, fiveCallback) {
   const values = [];
   // Add your code here
+  for (let i = startIndex; i <= stopIndex; i++) {
+    values.push[i];
+  }
+  values.forEach(value => {
+    if (value % 3 === 0 && value % 5 === 0) {
+      threeCallback(value);
+      fiveCallback();
```

Comment:
A minor problem here. You missed sending `value` as a parameter to the function. Also, I think you can also write the if block as if (values[i] % 15 === 0). If a number is both divisible by 3 and 5, it would mean that it would always be divisible by 15. This way you can avoid the extra && condition. What do you think?